src/_utils.ts

```diff
@@ -72,6 +65,34 @@ export function deserializeRaw(value: any) {
     // Return unknown strings as-is
     return value;
   }
-  checkBufferSupport();
-  return Buffer.from(value.slice(BASE64_PREFIX.length), "base64");
+  return genBytesFromBase64(value.slice(BASE64_PREFIX.length));
+}
+
+// Base64 utilities - Waiting for https://github.com/unjs/knitwork/pull/83 // TODO: Replace with knitwork imports as soon as PR is merged
```

Comment:
Can now be used via https://github.com/unjs/undio/pull/8

Reply:
Yes indeed. The feature is not releaed yet and I ned to complete node {Buffer} support but we shall adopt undio.

Reply:
will add a TODO to refactor once undio stable